applications/StructuralMechanicsApplication/custom_elements/small_displacement_mixed_volumetric_strain_element.cpp

```diff
@@ -8,6 +8,7 @@
 //
 //  Main authors:    Ruben Zorrilla
 //                   Riccardo Rossi
+//  Collaborator:    Vicente Mataix Ferrandiz
```

Comment:
We have spent 6 weeks developing, implementing and improving this formulation...

Reply:
I agree, sorry @loumalouomega  but i do not agree with this

Reply:
Removing it, in any case collaborator is not author